style-guide.md

```diff
@@ -0,0 +1,585 @@
+table of contents or anywhere else in the documentation.
+```
+
+reStructuredText allows you to use any non-alpha-numeric character for the
+underline adornment; however, pages must be internally consistent. The
+Archivematica documentation has not been particularly consistent across pages
+with the characters used for adornment in the past, but going forward we will
+try to use the following:
+
+* Title: === (overlined AND underlined)
+* First-level heading: ---
+* Second-level heading: ^^^
+* Third-level heading: +++
+* Fourth-level heading level: ###
+
+Note that the title should be overlined and underlined; section headings should
+only be underlined.
+
+#### Table of contents
+
+Include a table of contents after the page's introductory text block, linking to the anchor for each section. In reStructuredText, tables of contents are built by hand. The table of contents should be preceded by the italicized phrase _On this page_.
+
+Using the section anchors, create bulleted list containing an internal reference
+for each section that you would like to include in the table of contents using
+the reStructuredText link format.
+
+```
+.. _my-page:
+
+=============
+My Page Title
+=============
+
+Introductory page text goes here.
+
+*On this page*
+
+* :ref:`This is a first-level section <first-level>`
+* :ref:`This is another first-level section <another-first-level>`
+
+.. _first-level:
+
+This is a first-level section
+-----------------------------
+
+Information about this section. Note that this section has an anchor, since
+it is a first-level section. Using the anchor, we can link to it from the
+table of contents or anywhere else in the documentation.
+
+A second-level section
+^^^^^^^^^^^^^^^^^^^^^^
+
+Detailed information about this subsection. Note that this section does not
+have an anchor, so you cannot link to it from anywhere else.
+
+.. _another-first-level:
+
+This is another first-level section
+-----------------------------------
+
+Information about this section. Note that this section has an anchor, since
+it is a first-level section. Using the anchor, we can link to it from the
+table of contents or anywhere else in the documentation.
+
+```
+
+#### Text
+
+Text blocks should truncated after 80 characters.
+
+```
+Archivematica packages are hosted at packages.archivematica.org as a central
+place to store packages for multiple operating systems. Packages for both Ubuntu
+14.04 and 16.04 are available.
+```
+
+Code blocks are an exception to this rule. A line of code can extend beyond 80
+characters since a mid-line break may cause rendering errors.
+
+```
+.. code:: bash
+	 sudo sh -c 'echo "ARCHIVEMATICA_DASHBOARD_DASHBOARD_SEARCH_ENABLED=false" >> /etc/default/archivematica-dashboard'
+	 sudo sh -c 'echo "ARCHIVEMATICA_MCPSERVER_MCPSERVER_SEARCH_ENABLED=false" >> /etc/default/archivematica-mcp-server'
+	 sudo sh -c 'echo "ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_SEARCH_ENABLED=false" >> /etc/default/archivematica-mcp-client'
```

Comment:
You may want to use spaces instead of tabs. This happens in a few places in this doc.